runtime/src/dice.rs

```diff
@@ -52,3 +53,33 @@ fn cert_from_dccm(dv: &DataVault, cert: &mut [u8], cert_type: CertType) -> Calip
 
     Ok(size)
 }
+
```

Comment:
low cohesion

Reply:
Let me know if the new format addresses this at all. I didn't catch you on IM before I made this most recent change.

Reply:
It can be a follow up PR. Note you had to import several definitions - the mailbox API can reside in a different module rather than being included in several places. Handlers can be all in a single module.

Reply:
Ah okay I get what you are saying. I do like the idea of consolidating it. Maybe the three of us should do a quick sync on follow up changes this week. Stuff like that will be much easier to tweak since it won't really affect testing or anything as much compared to this change.